using-valtimo/process/process-beans/job-service.md

```diff
@@ -0,0 +1,23 @@
+# Job service
+
+This service provides a way to change Timer Events withing the current process
+
+### Changing Timer event date  
+
+Job service provides two methods that allow for an update on the timer due date in the current process
```

Comment:
What are these useful for? I think we should give an example.

Reply:
well i understand and agree but then again i really don't know a specific circumstance where this is useful.